pkgs/applications/window-managers/dwm/dwm-status.nix

```
@@ -1,31 +1,42 @@
 { stdenv, lib, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, makeWrapper, pkgconfig, xorg
-, enableAlsaUtils ? true, alsaUtils, bash, coreutils }:
+, enableAlsaUtils ? true, alsaUtils, coreutils
+, enableNetwork ? true, dnsutils, iproute, wirelesstools }:
 
 let
-  binPath = stdenv.lib.makeBinPath [
-    alsaUtils bash coreutils
-  ];
+  bins =
```

```suggestion
   bins = lib.optionals enableAlsaUtils [
     alsaUtils
     coreutils
   ] ++ (lib.optionals enableNetwork [
     dnsutils
     iproute
     wirelesstools
   ]);
```

(not sure if the parenthesis are necessary)

---

I will check it